.github/workflows/docker.yml

```diff
@@ -0,0 +1,45 @@
+name: "Docker Build"
+
+on:
+  push:
+    tags:
+      - '*'
+    branches:
+      - 'master'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: Get tag
+        id: repository
+        run: echo "tag=$(git describe --tags HEAD)" > $GITHUB_ENV
+
+      - name: Set up QEMU
```

Comment:
Why's QEMU being used here? (not sure about the specifics of python builds)

Reply:
To build for platforms other than amd64 qemu is needed first, I've added arm64 to the list of platforms for raspberry pi to be able to run the containers.

For reference it's in the [docker documentation for buildx actions](https://github.com/docker/setup-buildx-action#usage).

I should preserve the comments from the example.

Reply:
I added in comments for each step

Reply:
Thanks! Didn't know docker could do that, good to know